lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -104,7 +110,10 @@ export abstract class PerformanceClient implements IPerformanceClient {
             clientId: this.clientId,
             name: measureName,
             startTimeMs: Date.now(),
-            correlationId: eventCorrelationId
+            correlationId: eventCorrelationId,
+            idTokenSize: this.idTokenSize,
+            accessTokenSize: this.accessTokenSize,
+            refreshTokenSize: this.refreshTokenSize,
```

Comment:
These should be passed in on `endMeasurement`, not start.

Reply:
I would propose that instead of attaching the sizes as properties on the performanceclient, to instead include them as properties on whatever low-level submeasurement has access to the sizes, and then pull them out and attach them as top-level properties on teh performance event in `flushMeasurement`.

Reply:
For example:

```typescript
                const eventToEmit = sortedCompletedEvents.reduce((previous, current) => {
                    if (current.name !== measureName) {
                        this.logger.trace(`PerformanceClient: Complete submeasurement found for ${current.name}`, correlationId);
                        // TODO: Emit additional properties for each subMeasurement
                        const subMeasurementName = `${current.name}DurationMs`;
                        
                        /*
                         * Some code paths, such as resolving an authority, can occur multiple times.
                         * Only take the first measurement, since the second could be read from the cache,
                         * or due to the same correlation id being used for two distinct requests.
                         */
                        if (!previous[subMeasurementName]) {
                            previous[subMeasurementName] = current.durationMs;
                        } else {
                            this.logger.verbose(`PerformanceClient: Submeasurement for ${measureName} already exists for ${current.name}, ignoring`, correlationId);
                        }

                         const accessTokenSize = current.accessTokenSize;
                         if (!previous.accessTokenSize) {
                              previous.accessTokenSize = accessTokenSize;
                         }
                    }
                    return previous;
                }, topLevelEvent);

```